event-app-api/JWT.js

```diff
@@ -0,0 +1,33 @@
+import pkg from "jsonwebtoken";
+
+const { sign, verify } = pkg;
+export const createTokens = (result) => {
+  //generates an access token using the jsonwebtoken library
+  const accessToken = sign(
+    { username: result.username, id: result.id },
+    "ukpkitexmd",
```

Comment:
you should add an inline comment for ambiguous strings like this. What does the string represent and what is it used for?